### PR TITLE
Adding new NuSpec file with Microsoft.Bcl.Async dependency.

### PR DESCRIPTION
**Pull request by jgarverick**
Adding dependency to NuSpec file as per issue https://github.com/ahmetalpbalkan/Docker.DotNet/issues/18.  NuGet package will need to be rebuilt and deployed in order for changes to take effect.

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
LGTM will merge when I'll release 1.1.2.

**Comment by ahmetalpbalkan** [comment]
Why close?

**Comment by jgarverick** [comment]
My bad...didn't mean to do that. :/

File: Docker.DotNet/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```
Comment (ahmetalpbalkan) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: @jgarverick hey, I'm trying to understand why this change is needed. :smiley:  finally had time to come back to this repo and release a new version.

Comment (jgarverick) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: I'm not sure it matters. I think I initially it was because of the version of the API it was supporting.

-----Original Message-----

Comment (ahmetalpbalkan) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: @jgarverick hmm not really actually. we're doing that using the nuget pkg version. If you could revert only this file I'd be happy to merge right away.

Comment (jgarverick) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: I will try to get after it later...I'm currently waiting for my daughter to come out of surgery so it'll be a while before I can get to my laptop.

-----Original Message-----

Comment (ahmetalpbalkan) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: @jgarverick ok I guess I'll just merge like this and then take a fix. Thanks. I hope all goes well for your daughter. :heart:

Comment (jgarverick) [inline Docker.DotNet/Properties/AssemblyInfo.cs]: Thanks, man....appreciate it.

-----Original Message-----

**Comment by ahmetalpbalkan** [comment]
heh I just realized we have two branches `dev`/`master` which are pretty diverged from each other. :) I'm going to remove `dev` after integrating the changes over there.

**Comment by jgarverick** [comment]
Sounds good to me. 😊 

-----Original Message-----
